browser/ui/webui/settings/brave_social_blocking_handler.cc

```diff
@@ -0,0 +1,50 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/webui/settings/brave_social_blocking_handler.h"
+
+#include <string>
+
+#include "base/bind.h"
+#include "base/values.h"
+#include "brave/common/pref_names.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/common/pref_names.h"
+#include "components/prefs/pref_service.h"
+#include "content/public/browser/web_ui.h"
+#include "content/public/browser/web_ui_data_source.h"
+
+BraveSocialBlockingHandler::BraveSocialBlockingHandler()
```

Comment:
I'm don't understand the naming of this class. What google sign in for extensions have to do with social blocking?

Reply:
@bridiver - The pre-existing toggle for `Google Logins` in `brave://settings/socialBlocking` will also be used to enable/disable `Google Login` for extensions hence the name.

Reply:
but then why does this only handle one of the settings? Either this is a handler for all social blocking settings or its a handler for enabling google logins. I think either change the name or move all the settings in this handler